libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

```diff
@@ -70,6 +70,27 @@ Stack Monitoring UI. To get a cluster's `cluster_uuid`,
 call the `GET /` API against that cluster.
 <2> This setting identifies the hosts and port numbers of {es} nodes
 that are part of the monitoring cluster.
++
+If you want to use PKI authentication to send monitoring events to
+{es}, you must specify a different set of configuration options. For
+example:
++
+["source","yml",subs="attributes"]
+--------------------
+monitoring:
+  enabled: true
+  cluster_uuid: PRODUCTION_ES_CLUSTER_UUID <1>
```

Comment:
```suggestion
  cluster_uuid: PRODUCTION_ES_CLUSTER_UUID
```

Reply:
I find the callouts to be kind of confusing actually (and they don't render well in the HTML when you have multiple callouts with the same number), so I suggest removing them.